scripts/gulp/create-style-bundle.js

```diff
@@ -7,10 +7,17 @@ const autoprefixer = require('autoprefixer');
 const postcss = require('postcss');
 const sass = require('sass');
 
+/**
```

Comment:
These changes are copied from frontend-shared, but not required at this time. Its worth noting that this file is still not typechecked and the sibling file create-bundle.js is still outdated and needs more refactoring to sync it with frontend-shared. I would suggest this is a TODO for later.